pkg/resource/endpoint/custom_update_api.go

```diff
@@ -0,0 +1,123 @@
+package endpoint
```

Comment:
recommend merging this file with `hooks.go` and adding the standard license header...

Reply:
I created a tracking issue where all these things are captured (headers). Will create follow-up PR to fix.